.gitignore

```diff
@@ -24,3 +24,5 @@ carpincho_mq-*.tar
 
 # Temporary files, for example, from tests.
 /tmp/
+
+.idea/
```

Comment:
Estoy completamente desepcionado con esta linea Jajaja

Reply:
no sirve vscode, no tiene el go to method, ya con eso se convierte en inservible